safekeeper/spec/models/MCProposerAcceptorReconfig_p2_a2_t2_l2_c3.cfg

```diff
@@ -0,0 +1,18 @@
+CONSTANTS
+NULL = NULL
+proposers = {p1, p2}
+acceptors = {a1, a2}
+max_term = 2
+max_entries = 2
+max_generation = 3
+SPECIFICATION Spec
+CONSTRAINT StateConstraint
+INVARIANT
+TypeOk
+ElectionSafetyFull
+LogIsMonotonic
+LogSafety
+\* CommittedNotTruncated
```

Comment:
Why not? In theory this configration should never be committing anything because only 2 acceptors

Reply:
I think discussed verbally, but it does commit: this simplest model with 2 acceptors considers all possible memberships of these 2, e.g. {a1}. And it is perfectly fine to commit record after {a1} flushed it in the confiugration consisting only of {a1}, we have quorum. So the model does commits. This particular invariant is generally not expected to hold as its comment explains, but in such small model it is; added comment about that.